recipe/meta.yaml

```diff
@@ -1,33 +1,49 @@
-{% set name = "uharfbuzz" %}
 {% set version = "0.51.1" %}
 
 package:
-  name: {{ name|lower }}
+  name: uharfbuzz
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/uharfbuzz-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/u/uharfbuzz/uharfbuzz-{{ version }}.tar.gz
   sha256: d651174806fef320c6a036b0ba395ce8837549de26cc65cf18ae879012d2fba0
 
 build:
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  script:
+    - rm -rf harfbuzz           # [unix]
+    - export USE_SYSTEM_LIBS=1  # [unix]
+    - del harfbuzz /s /q        # [win]
+    - set USE_SYSTEM_LIBS=1     # [win]
+    # Make sure we can find the harfbuzz-subset package
+    # This command makes debugging the setup script much easier
+    - pkg-config --exists harfbuzz-subset --debug
+    - {{ PYTHON }} -m pip install . -vv
+  number: 1
+  ignore_run_exports_from:
+    - zlib
+    - glib
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
+    - pkgconfig                              # [build_platform != target_platform]
+    - pkg-config
     - {{ compiler('cxx') }}
-    - {{ stdlib("c") }}
-    - cython
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
   host:
     - pip
     - python
     - setuptools-scm
     - cython
     - pkgconfig
     - setuptools
+    - harfbuzz
+    # The following dependencies are needed for harfbuzz->freetype detection on windows
+    - zlib
+    - glib
```

Comment:
Should these dependencies be restricted to windows only?

Reply:
sure but we ignore run export on them for all libraries.

it isn't such a burden to depend on them.

Reply:
feel free to add the selector.